c_src/ep_node.c

```diff
@@ -724,6 +724,12 @@ stack_ensure_all(ErlNifEnv *env, ep_cache_t *cache)
                     spot--;
                     continue;
                 }
+                
+                // skip over enums -- can't be nested
+                if (spot->node->n_type == node_enum) {
```

Comment:
I'm a bit confused here – you want to skip over enums but the condition seems to be true when is is an enum? Shouldn't this be the other way around?

Reply:
oh, nevermind – I think I see. this avoids going into the following for loop if it is an enum. Right?

Reply:
yep!